### ISSUE TITLE
installer is marking units as executable, which is making systemd complain

### ISSUE THREAD

**Issue by Gooberpatrol66** [plattfot/baksnapper#3]
```
[ 4546.155464] systemd[1]: Configuration file /usr/lib/systemd/system/baksnapper@.timer is marked executable. Please remove executable permission bits. Proceeding anyway.
[ 4546.155586] systemd[1]: Configuration file /usr/lib/systemd/system/baksnapper@.service is marked executable. Please remove executable permission bits. Proceeding anyway.
[ 4546.155745] systemd[1]: Configuration file /usr/lib/systemd/system/baksnapper@.timer is marked executable. Please remove executable permission bits. Proceeding anyway.
[ 4546.155858] systemd[1]: Configuration file /usr/lib/systemd/system/baksnapper@.service is marked executable. Please remove executable permission bits. Proceeding anyway.

```

**Comment by plattfot** [comment]
Should be fixed in 1.0.2

**Comment by Gooberpatrol66** [comment]
yep thanks
